### ISSUE TITLE
Missing focus styles for accessibility

### ISSUE THREAD

**Issue by danielroe** [lusaxweb/vuesax-next#8]
These are beautiful but currently missing focus styles (particularly the checkbox, switch and radio components), which are absolutely required to make components inclusive and accessible.

I'm aware this is under development but this is also true of v3 of the library.

**Comment by luisDanielRoviraContreras** [comment]
thank you very much we will be more aware and add that to the mentioned components
